app/src/main/java/org/schabi/newpipe/util/StreamItemAdapter.java

```diff
@@ -171,6 +180,32 @@ private View getCustomView(final int position, final View view, final ViewGroup
         return convertView;
     }
 
+    /**
+     * @param position which primary stream to check.
+     * @return whether the primary stream at position has a secondary stream associated with it.
+     */
+    private boolean hasSecondaryStream(final int position) {
+        return secondaryStreams != null && secondaryStreams.get(position) != null;
+    }
+
+    /**
+     * @return if there are any video-only streams with no secondary stream associated with them.
+     * @see #hasVideoOnlyWithNoSecondaryStream
+     */
+    private boolean checkHasVideoOnlyWithNoSecondaryStream() {
```

Comment:
I would rename that method to ``checkHasAnyVideoOnlyStreamWithNoSecondaryStream``